parsons/google/google_sheets.py

```diff
@@ -32,12 +32,17 @@ def __init__(self, google_keyfile_dict=None):
 
         if google_keyfile_dict is None:
             try:
-                keyfile_json = os.environ['GOOGLE_DRIVE_CREDENTIALS']
+                google_drive_credentials = os.environ['GOOGLE_DRIVE_CREDENTIALS']
```

Comment:
For consistency and for optional handling of strings, I would recommend using this google app credential utility: https://github.com/move-coop/parsons/blob/master/parsons/google/utitities.py#L7-L20

Reply:
Thanks, I've modified the GoogleSheets connector to use that utility [here](https://github.com/move-coop/parsons/pull/618/commits/b2cc67cba7b66129b6e67ca4e03ec22587433d6f).

I wonder if using this utility might not be entirely backward compatible for Parsons users. It modifies the credntials environment variable, which is not something the GoogleSheets connector used to do.